kit/Kit.cpp

```diff
@@ -2576,6 +2577,17 @@ void lokit_main(
                     return false;
                 }
 
+                // Mount the directory for fonts downloaded from servers, if configured
+                const std::string jailTmpFontDir = Poco::Path(jailPath, tmpFontDir.substr(1)).toString();
+                LOG_INF("Mounting temp font dir " << tmpFontDir << " -> " << jailTmpFontDir);
+                if (!JailUtil::bind(tmpFontDir, jailTmpFontDir)
+                    || !JailUtil::remountReadonly(tmpFontDir, jailTmpFontDir))
```

Comment:
We worked quite hard to get the number of bind mounts per jail down to a small number - and need to do more work to get it lower still really. Is it really unavoidable that we have to have a new bind mount for fonts ?

Reply:
Well, the point is that I need to font files to be accessible using the same pathname both in the non-jailed ForKit process (which passes the font pathname on to the OutputDevice::AddTempDevFont() function) and in a jailed Kit process (which opens and mmaps the font file using the same pathname in the FreetypeFontFile::Map() function).

But sure, I can attempt to modify the core code so that I can pass a separate pathname to be used when the font file is opened later.

Reply:
Makes perfect sense - I can see how that would be easier but hoarding the bind-mounts makes sense. Then again - bind mounts are not really the right thing to do we should really be using: https://man7.org/linux/man-pages/man7/namespaces.7.html and (I think) creating a new namespace per kit process - particularly becase:

 Namespace lifetime
       Absent any other factors, a namespace is automatically torn down
       when the last process in the namespace terminates or leaves the
       namespace. 

Cleanup is much nicer, and it doesn't clutter your 'mount' command something horribly ;-) and possibly we could do these things hierarchically or something (or something ;-) really we need a container whizz to advise us - I've been meaning to talk to Lennart or someone at a FOSDEM in-person but not got to one recently.